exonum/src/helpers/fabric/maintenance.rs

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![allow(missing_debug_implementations)]
+
+//! This module implements node maintenance actions.
+// spell-checker:ignore exts
+
+use std::path::Path;
+use std::collections::HashMap;
+
+use blockchain::Schema;
+use helpers::config::ConfigFile;
+use storage::{Database, DbOptions, RocksDB};
+use node::NodeConfig;
+use super::internal::{CollectedCommand, Command, Feedback};
+use super::{Argument, CommandName, Context};
+
+const NODE_CONFIG_PATH: &str = "NODE_CONFIG_PATH";
+const DATABASE_PATH: &str = "DATABASE_PATH";
+const MAINTENANCE_ACTION_PATH: &str = "MAINTENANCE_ACTION_PATH";
+
+/// Maintenance command.
+/// Supported actions:
+/// * clear-cache - clear message cache.
+pub struct Maintenance;
+
+impl Maintenance {
+    /// Returns the name of the `Maintenance` command.
+    pub fn name() -> CommandName {
+        "maintenance"
+    }
+
+    fn node_config(ctx: &Context) -> NodeConfig {
+        let path = ctx.arg::<String>(NODE_CONFIG_PATH)
+            .expect(&format!("{} not found.", NODE_CONFIG_PATH));
+        ConfigFile::load(path).unwrap()
```

Comment:
Let's use `except` here - something can go wrong while loading and it's better to give user a bit more info rather than backtrace only.

Reply:
Those lines were simply copy-pasted from `Run::db_helper`. I guess it should be changed there too then :)